rest/taskana-rest-spring/src/test/java/pro/taskana/rest/security/SampleLoginModule.java

```diff
@@ -0,0 +1,84 @@
+package pro.taskana.rest.security;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.security.auth.Subject;
+import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.callback.NameCallback;
+import javax.security.auth.callback.PasswordCallback;
+import javax.security.auth.spi.LoginModule;
+
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+import pro.taskana.ldap.LdapCacheTestImpl;
+import pro.taskana.rest.resource.AccessIdResource;
+import pro.taskana.security.GroupPrincipal;
+import pro.taskana.security.UserPrincipal;
+
+/**
+ * TODO.
```

Comment:
Please make a real comment.

Reply:
Haha .... good catch. Copied from the common module.